src/pages/_app.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import '../styles/globals.css';
-import Header from '../components/header.js';
+import Header from '../components/header';
```

Comment:
Removed `.jsx` in accordance with the [imports/extensions](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) rule.